### PR TITLE
docs: Metrics storybook design

### LINKED ISSUE #1454: Create the Design section for Storybook feature
Description
Use feature template to create the Design section for the Storybook feature.

Acceptance Criteria

- [ ] create the Design section for Storybook feature

Additional context
[Storybook feature design](https://github.com/Flank/flank-dashboard/blob/master/metrics/web/docs/features/storybook/01_metrics_storybook_design.md)

### PR DESCRIPTION
**Pull request by solid-tetianamovlian**
Fixes #1454 

## Checklist

- [x] Documented

### CODE REVIEW

**Review by solid-dimakoniaiev — CHANGES_REQUESTED** [review]
Well done, @solid-maxim @solid-tetianamovlian! I've added a couple of suggestions below - please let me know what you think!

File: metrics/web/docs/features/storybook/diagrams/metrics_storybook_class_diagram.puml [hunk 1]
```diff
@@ -0,0 +1,148 @@
+@startuml metrics_storybook_class_diagram
+
+package widgets/theme {
+    class ThemeNotifier {
+        - _selectedTheme : ThemeType
+        + isDark : bool
+        + selectedTheme : ThemeType
```
Comment (solid-dimakoniaiev) [inline metrics/web/docs/features/storybook/diagrams/metrics_storybook_class_diagram.puml:7]: Please apply where necessary.

```suggestion
        - _selectedTheme: ThemeType
        + isDark: bool
        + selectedTheme: ThemeType
```

**Review by solid-vovabeloded — CHANGES_REQUESTED** [review]
Well done, @solid-tetianamovlian & @solid-maxim! I've added few suggestions here - please take a look.

**Comment by solid-maxim** [comment]
Close this PR for now as we want to improve the storybook design document.

**Pull request closed without merging**